exporter/clickhousemetricsexporter/exporter.go

```diff
@@ -84,7 +84,7 @@ func NewPrwExporter(cfg *Config, set component.ExporterCreateSettings) (*PrwExpo
 	}
 	ch, err := NewClickHouse(params)
 	if err != nil {
-		zap.S().Error("couldn't create instance of clickhouse")
+		log.Fatalf("Error creating clickhouse client: %v", err)
```

Comment:
No particular reason; I didn't know we could Fatal with sugar logger.

Reply:
ok, we should replace std logging library with sugar logger everywhere in the future PRs to standardize.

Reply:
We don't set up any logger config ourselves. The current zap.S() doesn't really log the message. That could be its own issue to address since there is a mixed use of `log`, `logrus` and `zap` in the codebase.